VK/VK/Animators/PushAnimator.swift

```diff
@@ -0,0 +1,55 @@
+// PushAnimator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// класс для анимирования перехода на другой экран вперед
```

Comment:
```suggestion
// Анимирование перехода на другой экран вперед
```
класс не пишем и так понятно